roles/ceph-osd/defaults/main.yml

```diff
@@ -4,15 +4,15 @@ ceph_osd:
   pkgs:
     - parted
 
-  # ONLY SET THIS TO TRUE IF YOU KNOW WHAT YOU'RE DOING.
-  # This will likely only be used on a fresh install.
-  destroy_devices_no_matter_what: false
+  # choose one scenario
+  journal_collocation: false

```

Comment:
On second thought, I wouldn't feel comfortable pushing the disk detection for `raw_multi_journal` so perhaps we should leave it at `journal_collocation` for now.